desktop/faqs.md

```diff
@@ -0,0 +1,215 @@
+
+{% include experimental.md %}
+
+### Where can I find information about diagnosing and troubleshooting Docker Desktop issues?
+
+You can find information about diagnosing and troubleshooting common issues in the Troubleshooting topic. See [Mac Logs and Troubleshooting](../docker-for-mac/troubleshoot.md) topic and Windows Logs and [Windows Logs and Troubleshooting](../docker-for-windows/troubleshoot.md).
+
+If you do not find a solution in Troubleshooting, browse issues on
+[docker/for-mac](https://github.com/docker/for-mac/issues){: target="_blank" rel="noopener" class="_"} or [docker/for-win](https://github.com/docker/for-win/issues){: target="_blank" rel="noopener" class="_"} GitHub repository, or create a new one.
+
+### How do I connect to the remote Docker Engine API?
+
+To connect to the remote Engine API, you might need to provide the location of the Engine API for Docker clients and development tools.
+
+Mac users can connect to the Docker Engine through a Unix socket: `unix:///var/run/docker.sock`.
+
+If you are working with applications like [Apache Maven](https://maven.apache.org/){: target="_blank" rel="noopener" class="_"}
+that expect settings for `DOCKER_HOST` and `DOCKER_CERT_PATH` environment
+variables, specify these to connect to Docker instances through Unix sockets.
+
+For example:
+
+```bash
+export DOCKER_HOST=unix:///var/run/docker.sock
+```
+
+Docker Desktop Windows users can connect to the Docker Engine through a **named pipe**: `npipe:////./pipe/docker_engine`, or **TCP socket** at this URL:
+`tcp://localhost:2375`.
+
+This sets `DOCKER_HOST` and `DOCKER_CERT_PATH` environment variables to the
+given values (for the named pipe or TCP socket, whichever you use).
+
+See also [Docker Engine API](../engine/api/index.md) and Docker Desktop for Mac forums topic
+[Using pycharm Docker plugin](https://forums.docker.com/t/using-pycharm-docker-plugin-with-docker-beta/8617){: target="_blank" rel="noopener" class="_"}.
+
+### How do I connect from a container to a service on the host?
+
+Both Mac and Windows have a changing IP address (or none if you have no network access). On both Mac and Windows, we recommend that you connect to the special DNS name `host.docker.internal`, which resolves to the internal IP address used by the host. This is for development purposes and does not work in a production environment outside of Docker Desktop.
+
+For more information and examples, see how to connect from a container to a service on the host
+[on Mac](../docker-for-mac/networking.md#i-want-to-connect-from-a-container-to-a-service-on-the-host) and [on Windows](../docker-for-windows/networking.md#i-want-to-connect-from-a-container-to-a-service-on-the-host).
+
+### How do I connect to a container from Mac or Windows?
+
+We recommend that you publish a port, or connect from another container. Port forwarding works for `localhost; --publish`, `-p`, or `-P` all work. Ports exposed from Linux are forwarded to the host.
```

Comment:
I think we should remove the last sentence, the FAQ entry is about Mac and Windows, I’m thinking it will confuse users what `Linux` means in this context.